docs/posts/ibis-version-9.0.0-release/index.qmd

```diff
@@ -0,0 +1,486 @@
+Day Unix backend](../unix-backend/index.qmd)! As always, there have been
+backend-specific improvements and bug fixes. Some highlights are below.
+
+Check the [full changelog](../../release_notes.qmd#9.0.0) for more details.
+
+If you're new to Ibis, see [how to install](../../../install.qmd) and [the
+getting started tutorial](../../../tutorials/getting_started.qmd).
+
+To follow along with this blog, ensure you're on `'ibis-framework>=9,<10'`.
+First, we’ll setup Ibis for interactive use:
+
+```{python}
+import ibis
+import ibis.selectors as s
+
+ibis.options.interactive = True
+ibis.options.repr.interactive.max_rows = 3
+```
+
+And fetch the penguins dataset:
+
+```{python}
+t = ibis.examples.penguins.fetch()
+t
+```
+
+### Snowflake
+
+You can now run Ibis inside of a Snowflake stored procedure! Look for a blog on
+this coming soon.
+
+### DuckDB
+
+In this release, `asof` joins are also now supported with a `tolerance`
+parameter.
+
+With [DuckDB's addition of MySQL as a database it can attach
+to](https://duckdb.org/2024/01/26/multi-database-support-in-duckdb.html), we've
+added a `read_mysql` function to Ibis inline with `read_postgres` and
+`read_sqlite`.
+
+You can now cast binary data to geometry in the DuckDB backend, enabling use of regular Parquet files:
+
+```{python}
+zones = ibis.examples.zones.fetch().relocate("geom")  # <1>
+zones.to_parquet("zones.parquet")  # <2>
+```
+
+1. Fetch the zones example data with the geometry column.
+2. Write to a Parquet file, converting the geometry column to binary.
+
+```{python}
+zones = ibis.read_parquet("zones.parquet")  # <1>
+zones.schema()  # <2>
+```
+
+1. Load in the zones Paruqet file.
+2. Notice the geometry column is binary.
+
+```{python}
+zones = zones.mutate(geom=zones["geom"].cast("geometry"))  # <1>
+zones.schema()  # <2>
+```
+
+1. Cast the binary geometry column to a geometry column.
+2. Notice the geometry column is now a geometry type.
+
+### PySpark
+
+PySpark 3.5 is now supported.
+
+### Exasol
+
+Numerous improvements have been made to the Exasol backend by Nicola including
+bit operations, time operations, and hexdigest.
+
+### Apache Flink
+
+Scalar user-defined functions (UDFs) are now supported in the Flink backend.
+
+### RisingWave
+
+Streaming data definition language (DDL) support has been added to the
+RisingWave backend.
+
+## Breaking changes
+
+You can view the [full changelog](../../release_notes.qmd#9.0.0) for additional
+breaking changes. There is [one we expect to affect many
+users](#what-does-schema-mean).
+
+### What does "schema" mean?
+
+In Ibis, a "schema" is a mapping of column names to their types:
+
+```{python}
+t.schema()
+```
+
+And as of Ibis 9.0, that is all "schema" means! In building a standard Python
+dataframe interface, we must standardize naming conventions and terminology
+across data systems. Many systems use the words "schema", "database", "catalog",
+and "namespace" to mean different things.
```

Comment:
Maybe swap the "schema can mean other things" sentences and the "And as of ..." sentences. In this order it's a bit confusing if you're not familiar with other users of "schema".

Reply:
moved things around